### PR TITLE
Feat

### PR DESCRIPTION
**Pull request by scitoo**
A ordinary pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # Patela
+
+Update #1
```
Comment (profmarciotorres) [inline README.md]: Change `#` to `no.`

**Review by profmarciotorres — CHANGES_REQUESTED** [review]
Make that change, pls

File: README.md (hunk 1 above)
Comment (scitoo) [reply README.md]: Why?

Comment (profmarciotorres) [reply README.md]: Why I said so.

**Pull request merged**